tokio/src/runtime/handle.rs

```diff
@@ -201,6 +204,105 @@ impl Handle {
         let _ = self.blocking_spawner.spawn(task, &self);
         handle
     }
+
+    /// Run a future to completion on the Tokio runtime. This is the
+    /// runtime's entry point.
```

Comment:
This is incorrect, this is not the entry point but just an additional `block_on`?